File: 004_first_and_last/first.rb
#!/usr/bin/env ruby

ARGF.each do |line|
    num = line.to_i
    s   = num ** 2
    
    if s > 9
        s = s.to_s
        first = s[0].to_i
        last  = s[-1].to_i
        sum = first + last
        puts sum
    else
        puts s
    end
end

File: 004_first_and_last/first.py
#!/usr/bin/python

import sys

for line in sys.stdin:

  num = int(line)
  square = num ** 2

  if square > 9:

    square = str(square)
    first = int(square[0])
    last  = int(square[-1])
    sum = first + last
    print sum
  else:

    print square 
